logic/StateManager.py

```diff
@@ -28,46 +28,58 @@ def load_state(self, path: str) -> None:
         """Load state from file."""
         file = open(path, mode='r')
         self.state = json.load(file)
+        self.set_state()
         file.close()
 
     def reset(self):
         """Reset everything"""
         self.game.situation_manager.reset()
-        self.player.stats.reset()
-        self.game.state_manager.load_initial_state()
+        self.player_stats.reset()
+        self.load_initial_state()
         self.game.situation_manager.load_situation()
         self.game.game_screen.update_text()
 
     def set_state(self):
         """Sets the player state to the state that is loaded"""
-        cur_stats = self.player.stats
-        cur_stats.athletic_ability = self.state[0]['player']['stats']['athletic ability']
-        cur_stats.charisma = self.state[0]['player']['stats']['charisma']
-        cur_stats.smartness = self.state[0]['player']['stats']['smartness']
-        cur_stats.wisdom = self.state[0]['player']['stats']['wisdom']
-        self.player.stats.update_text()
+        self.player_stats.sus_int, self.player_stats.sad_int = 0, 0
+        for x in self.state[0]['player']['stats']:
```

Comment:
use 
`for keys, values in self.state[0]['player']['stats']:`

Reply:
I did that